modules/ROOT/pages/dataweave-language-introduction.adoc

```diff
@@ -257,6 +257,104 @@ Now the output meets the requirements of XML, so when you change the output dire
 </myroot>
 ----
 
+== Escaping Special Characters
+
+In DataWeave, you use the backslash (`\`) to escape special characters that
+you are using in an input string.
+
+There are only a few special characters within a string:
+
+* `$`: You need to escape any use of of `$` in a string. Otherwise, DataWeave
+treats the `$` as an unnamed parameter for a function and returns the error
+`Unable to resolve reference of $.`.
+* `"`: For a string that is surrounded by double quotes, you need to escape any
+double quote that is part of the string, for example, `"a\"bcdef"`. Here, the
+second double-quote is part of the string that starts with an `a` and ends
+with `f`.
+* `'` For a string that is surrounded by single quotes, you need to escape any
```

Comment:
Missing that \ also needs to be escaped. As is the escaping character.

Reply:
ah, thx, will do

Reply:
Also we should clarify that new line is written as \n a tab \t and unicode character can be written as \u<charcode> example \u25c4